fafbot.py

```diff
@@ -27,332 +30,54 @@
 import time
 from PySide import QtSql
 
-from fractions import Fraction
-from twitch import *
-import json
-
 from passwords import DB_SERVER, DB_PORT, DB_LOGIN, DB_PASSWORD, DB_TABLE
 
 from configobj import ConfigObj
 config = ConfigObj("/etc/faforever/faforever.conf")
 fafbot_config = ConfigObj("fafbot.conf")['fafbot']
 
-from threading import Timer
-
-TWITCH_STREAMS = "https://api.twitch.tv/kraken/streams/?game=" #add the game name at the end of the link (space = "+", eg: Game+Name)
-STREAMER_INFO  = "https://api.twitch.tv/kraken/streams/" #add streamer name at the end of the link
-GAME = "Supreme+Commander:+Forged+Alliance"
-HITBOX_STREAMS = "https://www.hitbox.tv/api/media/live/list?filter=popular&game=811&hiddenOnly=false&limit=30&liveonly=true&media=true"
-
-class betmatch(object):
-    def __init__(self, uid, startTime, name, odds, mostProbableWinner):
-        try:
-            self.uid = uid
-            self.startTime = startTime
-            self.pot = 0
-            self.players = {}
-            self.betOn = {}
-            self.name = name
-            self.odds = odds
-            self.mostProbableWinner = mostProbableWinner
-        except:
-            pass
-            
-
-    def computeReward(self, amount):
-        try:
-            maxTime = (30.0 * 60.0)
-            minTime = (2 * 60)
-            if (time.time() - self.startTime) > minTime:
-                return amount*(1.0-((time.time() - self.startTime) / maxTime))
-            else:
-                return amount
-        except:
-            return 0
-
-    def addBeter(self, amount, uid, betUid):
-        try:
-            maxBet = self.computeReward(amount)
-            if not uid in self.players :
-                self.players[uid] = maxBet
-            if not betUid in self.betOn:
-                self.betOn[betUid] =  []
-            self.betOn[betUid].append(uid)
-            return maxBet
-        except:
-            return 0
-
-    def getBetWinners(self, uid):
-        try:
-            if uid in self.betOn :
-                return self.betOn[uid]
-            else:
-                return []
-        except:
-            return []
-
-    def getReward(self, uid):
-        try:
-            if uid in self.players:
-                return self.players[uid]
-            return 0
-        except:
-            return 0
-
-class bettingSystem(object):
-    def __init__(self):
-        self.matches = {}
 
-    def addMatch(self, match):
-        try:
-            if not match.uid in self.matches:
-                self.matches[match.uid] = match
-            return self.matches[match.uid]
-        except:
-            return None
+def fetch_url_contents(url):
```

Comment:
Going to move this into MessageHandler.py soon, since it no longer makes sense to have it here after splitting out MessageRouter so the other dependencies will stay in MessageHandler.py anyway.